app/controllers/concerns/error_extender.rb

```diff
@@ -27,7 +27,7 @@ def render_jsonapi_internal_server_error(exception)
   end
 
   def render_jsonapi_bad_request(exception)
-    error = { status: '400', title: Rack::Utils::HTTP_STATUS_CODES[400] }
+    error = { status: '400', title: Rack::Utils::HTTP_STATUS_CODES[400], detail: exception.to_s }
```

Comment:
not blocking: could dry up the error object formatting to a private message
```
  def error_object(status, exception_message)
    { 
      status: status_code.to_s, 
      title: Rack::Utils::HTTP_STATUS_CODES[status_code], 
      detail: exception_message.to_s
    }
  end
```

Reply:
Done 👍